kpi/migrations/0060_commonexporttask_remove_projectviewexporttask_data_and_more.py

```diff
@@ -0,0 +1,169 @@
+
+import kpi.fields.file
+import kpi.fields.kpi_uid
+import kpi.models.asset_file
+import kpi.models.import_export_task
+
+
+def populate_common_export_tasks(apps, schema_editor):
+    CommonExportTask = apps.get_model('kpi', 'CommonExportTask')
+    ProjectViewExportTask = apps.get_model('kpi', 'ProjectViewExportTask')
+    for project_view_task in ProjectViewExportTask.objects.all():
+        common_task = CommonExportTask.objects.create(
+            data=project_view_task.data,
+            messages=project_view_task.messages,
+            status=project_view_task.status,
+            date_created=project_view_task.date_created,
+            result=project_view_task.result,
+            user=project_view_task.user,
+        )
+        project_view_task.commonexporttask_ptr = common_task
+        project_view_task.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('kpi', '0059_assetexportsettings_date_created_and_more'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CommonExportTask',
+            fields=[
+                (
+                    'id',
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name='ID',
+                    ),
+                ),
+                ('data', models.JSONField()),
+                ('messages', models.JSONField(default=dict)),
+                (
+                    'status',
+                    models.CharField(
+                        choices=[
+                            ('created', 'created'),
+                            ('processing', 'processing'),
+                            ('error', 'error'),
+                            ('complete', 'complete'),
+                        ],
+                        default='created',
+                        max_length=32,
+                    ),
+                ),
+                ('date_created', models.DateTimeField(auto_now_add=True)),
+                (
+                    'result',
+                    private_storage.fields.PrivateFileField(
+                        max_length=380,
+                        storage=(
+                            private_storage.storage.files.PrivateFileSystemStorage()
+                        ),
+                        upload_to=kpi.models.import_export_task.export_upload_to,
+                    ),
+                ),
+                (
+                    'user',
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to=settings.AUTH_USER_MODEL,
+                    ),
+                ),
+            ],
+            options={
+                'abstract': False,
+            },
+        ),
+        migrations.RunPython(populate_common_export_tasks),
```

Comment:
Forgot: if we end up keeping the migration, this needs to be reversible, even if the reverse is RunPython.noop. Otherwise unapplying migrations breaks